### DISCUSSION TITLE
Pre-trained Weights of AutoEncoder

### DISCUSSION THREAD

**Discussion by JustinYuu** [archinetai/audio-diffusion-pytorch#30] [Q&A]
Hi, it seems that the checkpoint files in [huggingface](https://huggingface.co/archinetai/audio-diffusion-pytorch/tree/main) are all used for class Model1d, I wonder if there exist any checkpoints available for the DiffusionAutoencoder1d class to perform the latent encoding?

**Comment by flavioschneider** [comment]
Not for now.  Moving this to the discussion section. Please use that for non-issues.
